### PR TITLE
Update the CLI version and changelog

### PR DESCRIPTION
**Pull request by Vidya2606**
This PR is created to update the CLI version and changelog.md for releasing to the marketplace.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,15 @@ Thank you @Eneuman,@Tatsinnit, @BartNetJS and @cxznmhdcxz for great contribution
...
 - [Change to service.name to show correct ingress url's](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/72)
 
+## [2.0.120231128]
+- Updating to latest B2K CLI version(will update the link)
+- [Update deprecated packages](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/64)
+- [Update support debugging with multi container pods](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/71)
+- [Added Unit tests to increase the code coverage](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/73)
+- [Remove the logged telemetry events which aren't used much](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/74)
+- [Support for downloading arm/arm64 binaries, Update minimum kubectl version to 1.27.3](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/76)
+- Address [Updated the unit tests for expected kubectl minimum version](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/78)
```
Comment (hsubramanianaks) [inline CHANGELOG.md:24]: Please remove the address word here.

**Review by hsubramanianaks — APPROVED** [review]
looks good to me.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,15 @@ Thank you @Eneuman,@Tatsinnit, @BartNetJS and @cxznmhdcxz for great contribution
 - [Update support doc with correct links](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/69)
 - [Change to service.name to show correct ingress url's](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/72)
 
+## [2.0.120231128]
```
Comment (Tatsinnit) [inline CHANGELOG.md:17]: 🐛  Why is this not at the top of this `changelog`? I think auto release will not detect this. 🥷❤️ cc: @hsubramanianaks  and @Vidya2606

Comment (hsubramanianaks) [reply CHANGELOG.md:17]: good one @Tatsinnit we missed it.

**Review by Tatsinnit — APPROVED** [review]
❤️🎊

**Pull request merged**